source/kraken/configuration.cpp

```diff
@@ -165,6 +167,10 @@ bool Configuration::is_realtime_add_enabled() const{
     return this->vm["GENERAL.is_realtime_add_enabled"].as<bool>();
```

Comment:
It would probably be safer to (untested):
```c++
return (is_realtime_enabled() && this->vm["GENERAL.is_realtime_add_enabled"].as<bool>());
```

Reply:
In this part of code, we simply load params from config. We don't want to insert logic. Only load the _raw value_ of the flag.

Reply:
I agree with Benoit, for the sake of consistency ! I would rather see the logic in the code, not the configuration. It would give us more flexibility.